### ISSUE TITLE
Improving a11y for inaccessible writing conventions

### ISSUE THREAD

**Issue by StephDriver** [BirkbeckCTP/janeway#4243]
### The observed issue:

There are conventions of layout from print that do not translate into accessible copy on webpages.  At heart there seem to be three different areas that give rise to these, sometimes in combination:  
- Some because it relies on visual layout for semantics (e.g. the position of an author's name in relation to the article title indicating that this is the author) and 
- others because they make use of typographical marks to convey information

### Consider
- [ ] How should we approach these - programmatically adapt them to a more accessible format? Require authors to avoid them?

### Examples
1.  [using `(1)` and indentation](https://www.glossa-journal.org/article/id/10082/) to indicate a worked example or figure.   These need to be marked up so that a screen-reader announces them as 'example 1 a' and later 'example 1 b', and they should be part of the heading structure.

<img width="513" alt="image" src="https://github.com/BirkbeckCTP/janeway/assets/5330770/6573e2fb-eedc-42ed-91bb-7b30a38e0840">
Here the xml has the whole example as a `<list list-type="gloss">` rather than putting the examples in the general article structure and using the gloss type for the details after the identifiers 1a and 1b.

2. Numbered section headings are read out as just the number, which is fine until the heading text also begins with number or could follow a number.  for example: "2. Tone in Akan" is read indistinguishably from "Two Tone in Akan" - this highlights the importance that section numbers are identified as section numbers.  Eg "Section 2.  Tone in Akan".

3. Use of Small Caps (or other visual formatting) to highlight a word or phrase.  This is purely visual so is ignored by screen readers, and the use of capitals for text is inaccessible for many who are reading the screen themselves.  it is particularly bad when a 'normal' capital is used for the start of the word, then small caps for the rest, as this breaks the word such that a screen reader will read the first letter and then try to read the rest of the word as if this was a separate word.

4. semantic use of small typographical marks, e.g. for example `27` to be compared with related example `27'`. This is a convention for many when having a related example, but that mark is too small to be an accessible difference for such an important distinction.

5. use of `[]` within quotations to show the inserted parts doesn't come across in screenreaders, and when it is used for part of a word, it interrupts that words so that the screenreader cannot read it correctly e.g. `[s]emantically, the` is read as `ess emantically, the`.
